### ISSUE TITLE
Modifying keybindings from within a module won't work.

### ISSUE THREAD

**Issue by maxim-uvarov** [nushell/nushell#13636]
### Describe the bug

I tried to modify the keybindings for some commands, but it didn't work. It seems like a minor bug.

### How to reproduce

```nu
export def --env 'test-keybinding' [] {
    $env.config.keybindings ++= [ {
        name: test
        modifier: control_alt
        keycode: char_j
        mode: [emacs, vi_normal, vi_insert]
        event: {
            send: executehostcommand
            cmd: 'ls'
        }
    } ]
}

# I use the module
> test-keybinding

# I try the shortcut here `ctrl+alt+j` with no luck

# While it shows that keybinding has been added
> $env.config.keybindings | last | to yaml
name: test
modifier: control_alt
keycode: char_j
mode:
- emacs
- vi_normal
- vi_insert
event:
  send: executehostcommand
  cmd: ls
```

### Expected behavior

I expect nu to apply the modified keybinding

### Configuration

| key                | value                                    |
| ------------------ | ---------------------------------------- |
| version            | 0.96.1                                   |
| major              | 0                                        |
| minor              | 96                                       |
| patch              | 1                                        |
| branch             |                                          |
| commit_hash        |                                          |
| build_os           | macos-aarch64                            |
| build_target       | aarch64-apple-darwin                     |
| rust_version       | rustc 1.78.0 (9b00956e5 2024-04-29)      |
| rust_channel       | stable-aarch64-apple-darwin              |
| cargo_version      | cargo 1.78.0 (54d8815d0 2024-03-26)      |
| build_time         | 2024-07-30 04:29:00 +00:00               |
| build_rust_channel | release                                  |
| allocator          | mimalloc                                 |
| features           | default, sqlite, system-clipboard, trash |
| installed_plugins  | polars 0.96.1, regex                     |

**Comment by NotTheDr01ds** [comment]
I'm not able to reproduce - Your example is working for me on Linux 0.96.1.

Out of curiosity, try dropping the `export` - You haven't actually created a module and don't need to `export` anything.  To actually do it in a module, you'd need to have:

```nu
module kbs {
    export def --env 'test-keybinding' [] {
        $env.config.keybindings ++= [ {
            name: test
            modifier: control_alt
            keycode: char_j
            mode: [emacs, vi_normal, vi_insert]
            event: {
                send: executehostcommand
                cmd: 'ls'
            }
        } ]
    }
}
use kbs
kbs test-keybinding 
```

**Comment by maxim-uvarov** [comment]
@NotTheDr01ds  thank you for checking! 

Nope, dropping export didn't help. Just in case I tried it kitty and wezterm.


<img width="544" alt="image" src="https://github.com/user-attachments/assets/4d650d11-4708-48b7-9118-92c43dcde711">

<img width="575" alt="image" src="https://github.com/user-attachments/assets/d380325d-6746-41e4-8808-4f52193a0136">

Executing in REPL works as expected:
<img width="562" alt="image" src="https://github.com/user-attachments/assets/1bf523aa-242a-4166-afa7-14f05cf2922f">

**Comment by maxim-uvarov** [comment]
Seems like it is fixed in 0.100.0! Woohoo! 
Thanks everyone!
